sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -4,15 +4,113 @@
 using System;
 using Azure.Core;
 using Azure.Identity;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
 
 namespace Azure.Communication.Sms.Tests.samples
 {
     /// <summary>
-    /// Samples that are used in the README.md file.
+    /// Samples to send sms
     /// </summary>
     public partial class Sample1_SmsClient
     {
-        public SmsClient CreateSmsClient()
+        [Test]
+        [SyncOnly]
+        public void SendSms()
+        {
+            SmsClient smsClient = CreateSmsClient();
+
+            #region Snippet:SendSms
+            /// Send an sms
+            SmsSendResult sendSmsResult = smsClient.Send(
+                from: "<leased-phone-number>",
+                to: "<to-phone-number>",
+                message: "<message-to-send>");
+
+            Console.WriteLine("MessageId: " + sendSmsResult.MessageId);
```

Comment:
`Console.WriteLine($"MessageId: {sendSmsResult.MessageId}");`

Let's use string interpolations in other snippets as well